.changeset/slimy-icons-grin.md

```diff
@@ -0,0 +1,8 @@
+---
+"@kaizen/components": patch
+---
+
+Clean up @kaizen/components.
+
+- Add `isAnimated` prop to `ErrorPage` for Chromatic compatibility
+- Rename directories to match convention
```

Comment:
The build would fail if you messed any of the naming/moving up. So I'm comfortable with killing the changeset

Reply:
Well the changeset is needed for the other prop now :P Or maybe still no since it only exists for our testing? 🤔